go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/osbuild/images
 
-go 1.21.0
+go 1.22.0
```

Comment:
This second commit reverts the revert in the go.mod.

Reply:
Wait what? Uh.